examples/naive.boa

```diff
@@ -0,0 +1,172 @@
+p: Project = input;
```

Comment:
Move this to 'test/known-good/' ? We dont have an examples directory and if you are going to put code examples in there, might as well use them as test cases.